pallets/phala/src/compute/base_pool.rs

```diff
@@ -96,14 +96,6 @@ pub mod pallet {
 		NftId,
 	);
 
-	type ShareTransferProxy<T> = (
```

Comment:
(Low priority) Please mention the change on this in the PR description.

Reply:
done